.github/workflows/kind.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
           IMG=flotta-operator:latest make docker-build
           kind load docker-image flotta-operator:latest
           timeout 2m make deploy IMG=flotta-operator TARGET=kind || retVal=$?
-          if [ $retVal -ne 124 ]; then
+          if [[ -n $retVal && $retVal -ne 124 ]]; then
```

Comment:
I'm sorry to bother you again on this but can you please check if `-n` works with non-string?
Since `-n` checks if a string is non-empty, I think we should use `-n "$retVal"` instead of `-n $retVal`.
(I'm still refering on [Bash conditional expression](https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html))

Reply:
`-n` is the opposite of `-z` which is used similarly [here](https://github.com/project-flotta/flotta-scale-tests/blob/main/scripts/run_test_plan.sh#L79), but I changed this to `-n "$retVal"` and it works too if you prefer :)